platformio.ini

```diff
@@ -408,6 +408,23 @@ build_flags =
   '-DLED_SEND_RECEIVE_ON=0'
   '-DGateway_Name="OpenMQTTGateway_ESP32_BLE"'
 
+[env:esp32dev-ble-mqtt-decode]
+platform = ${com.esp32_platform}
+board = esp32dev
+board_build.partitions = min_spiffs.csv
+lib_deps =
+  ${com-esp.lib_deps}
+  ${libraries.wifimanager32}
+  ${libraries.ble}
+  ${libraries.decoder}
+build_flags =
+  ${com-esp.build_flags}
+  '-DZgatewayBT="BT"'
+  '-DLED_SEND_RECEIVE=2'
+  '-DLED_SEND_RECEIVE_ON=0'
+  '-DGateway_Name="OpenMQTTGateway_ESP32_BLE"'
+  '-DMQTTDecodeTopic="MQTTDecode"'
```

Comment:
@h2zero could you replace `MQTTDecode` by `BTtoMQTT/undecoded`
And add also this macro into config_BT.h

Reply:
Done